src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/tasks/ImportReportsDataTask.kt

```diff
@@ -18,10 +18,10 @@ class ImportReportsDataTask(
   override fun performTask() {
     timeSource.yesterday().run {
       if (backDateEnabled) {
-        val importDate = service.dateOfLastImport()?.plusDays(1) ?: this
```

Comment:
This is the culprit.  Should not be adding anything here due to the actual reports being offset by a day anyway.  What this means is if the last import was on the 17th then the actual reports imported would be for the 16th.  So when we recover we would start on the 17th, the last date of a successful import.